app/error_handlers.py

```diff
@@ -30,4 +30,20 @@ def method_not_allowed(e):
 
 @app.errorhandler(500)
 def internal_error(e):
-    return make_response(jsonify(status=500, error='Internal Server Error', message='Well, this is embarrassing...'), status.HTTP_500_INTERNAL_SERVER_ERROR)
+	return make_response(jsonify(status=500, error='Internal Server Error', message='Well, this is embarrassing...'), status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+class InvalidPaymentID(Exception):
+    status_code = 404
```

Comment:
there's no test coverage for these functions

also, i don't think we need a status_code class variable if you're setting an instance variable of the same name via passed parameters